src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -49,58 +48,61 @@ public static CompletionHelper GetHelper(Document document)
         }
 
         public ImmutableArray<TextSpan> GetHighlightedSpans(
-            CompletionItem completionItem, string filterText, CultureInfo culture)
+            string text, string pattern, CultureInfo culture)
         {
-            var match = GetMatch(completionItem, filterText, includeMatchSpans: true, culture: culture);
+            var match = GetMatch(text, pattern, includeMatchSpans: true, culture: culture);
             return match == null ? ImmutableArray<TextSpan>.Empty : match.Value.MatchedSpans;
         }
 
         /// <summary>
-        /// Returns true if the completion item matches the filter text typed so far.  Returns 'true'
+        /// Returns true if the completion item matches the pattern so far.  Returns 'true'
         /// iff the completion item matches and should be included in the filtered completion
         /// results, or false if it should not be.
         /// </summary>
-        public bool MatchesFilterText(CompletionItem item, string filterText, CultureInfo culture)
-        {
-            return GetMatch(item, filterText, culture) != null;
-        }
+        public bool MatchesPattern(string text, string pattern, CultureInfo culture)
+            => GetMatch(text, pattern, culture) != null;
 
-        private PatternMatch? GetMatch(CompletionItem item, string filterText, CultureInfo culture)
-        {
-            return GetMatch(item, filterText, includeMatchSpans: false, culture: culture);
-        }
+        private PatternMatch? GetMatch(string text, string pattern, CultureInfo culture)
+            => GetMatch(text, pattern, includeMatchSpans: false, culture: culture);
 
         private PatternMatch? GetMatch(
-            CompletionItem item, string filterText,
+            string completionItemText, string pattern,
```

Comment:
How come the private helper names it `completionItemText`, but everything else uses just `text`.